### PR TITLE
Apply one-sentence-per-line in all adocs

### LINKED ISSUE #449: Use one sentence per line
Currently we have in our documentation multiple sentences in one line, like:
`The only thing that cannot be defined in the template itself is the unique ID of the submodel instance itself (it is not identical to the ID of the submodel template), as well as the property values, etc. Templates also define cardinalities, for example whether an element is optional or not. Submodel element lists typically contain more than one element: the template contains an exemplary element template; the other elements can be created by copy/paste from this template.`

There is a [recommendation from AsciiDoc community ](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line)to handle the text as a source code and to use one sentence per line.

They list the following advantages by using this technique:
```
- It prevents reflows (meaning a change early in the paragraph won’t cause the remaining lines in the paragraph to reposition).
- You can easily swap sentences.
- You can easily separate or join paragraphs.
- You can comment out sentences or add commentary to them.
- You can spot sentences which are too long or sentences that vary widely in length.
- You can spot redundant (and thus mundane) patterns in your writing.
```
- Additionally we will get much more clear git diffs

Proposal: Convert current documents to the `one sentence per line`-format and enforce this standard in all future documents

### PR DESCRIPTION
**Pull request by zrgt**
Fix #449

### CODE REVIEW

File: documentation/IDTA-01001/modules/ROOT/pages/annex/backus-naur-form.adoc [hunk 1]
```diff
@@ -7,14 +7,17 @@ https://creativecommons.org/licenses/by/4.0/).
 SPDX-License-Identifier: CC-BY-4.0
 
 Illustrations:
-Plattform Industrie 4.0; Anna Salari, Publik. Agentur für Kommunikation GmbH, designed by Publik. Agentur für Kommunikation GmbH
+Plattform Industrie 4.0; Anna Salari, Publik.
+Agentur für Kommunikation GmbH, designed by Publik.
+Agentur für Kommunikation GmbH
```
Comment (BirgitBoss) [inline documentation/IDTA-01001/modules/ROOT/pages/annex/backus-naur-form.adoc:12]: This split seems wrong

File: documentation/IDTA-01001/modules/ROOT/pages/annex/concepts-aas.adoc [hunk 2]
```diff
@@ -2,20 +2,26 @@
...
-https://creativecommons.org/licenses/by/4.0/). 
+https://creativecommons.org/licenses/by/4.0/).
 
 SPDX-License-Identifier: CC-BY-4.0
 
 Illustrations:
-Plattform Industrie 4.0; Anna Salari, Publik. Agentur für Kommunikation GmbH, designed by Publik. Agentur für Kommunikation GmbH
+Plattform Industrie 4.0; Anna Salari, Publik.
+Agentur für Kommunikation GmbH, designed by Publik.
+Agentur für Kommunikation GmbH
```
Comment (BirgitBoss) [inline documentation/IDTA-01001/modules/ROOT/pages/annex/concepts-aas.adoc:12]: See above

File: documentation/IDTA-01001/modules/ROOT/pages/annex/concepts-aas.adoc [hunk 3]
```diff
@@ -30,102 +36,135 @@ The following documents were used to identify requirements and concepts for the
...
+This mechanism makes it possible to convey semantically well-defined information to the Administration Shell.
 
 ====
-Note: Industry 4.0 and smart manufacturing in general will require many properties, which are beyond the current scope of IEC CDD, ECLASS, or other repositories. It is expected that these sets of properties will be introduced, as more and more domains are modelled and standardized (see next clause).
+Note: Industry 4.0 and smart manufacturing in general will require many properties, which are beyond the current scope of IEC CDD, ECLASS, or other repositories.
+It is expected that these sets of properties will be introduced, as more and more domains are modelled and standardized (see next clause).
 ====
 
-
 == The Concept of Submodels
```
Comment (BirgitBoss) [inline documentation/IDTA-01001/modules/ROOT/pages/annex/concepts-aas.adoc:94]: The space before == seems wrong?

File: documentation/IDTA-01001/modules/ROOT/pages/annex/concepts-aas.adoc [hunk 4]
```diff
@@ -30,102 +36,135 @@ The following documents were used to identify requirements and concepts for the
...
+For example, it will be possible to find a drilling machine by searching for an Administration Shell containing a submodel "Drilling" with appropriate properties.
+Certain properties can then be assumed to exist for communication between different I4.0 components.
+In our example, a second submodel "energy efficiency" could make sure the drilling machine is able to cut its electricity consumption when out of operation.
 
 ====
 Note: a side benefit of the Administration Shell will be to simplify the update of properties from product design (and in particular system design) tools, update properties from real data collected in the instances of assets, improve traceability of assets along the life cycle, and help certify assets from data.
 ====
 
-
 == Basic Structure of the Asset Administration Shell
```
Comment (BirgitBoss) [inline documentation/IDTA-01001/modules/ROOT/pages/annex/concepts-aas.adoc:119]: Agin a space?

File: documentation/IDTA-01001/modules/ROOT/pages/annex/handling-constraints.adoc [hunk 5]
```diff
@@ -7,15 +7,19 @@ https://creativecommons.org/licenses/by/4.0/).
 SPDX-License-Identifier: CC-BY-4.0
 
 Illustrations:
-Plattform Industrie 4.0; Anna Salari, Publik. Agentur für Kommunikation GmbH, designed by Publik. Agentur für Kommunikation GmbH
+Plattform Industrie 4.0; Anna Salari, Publik.
+Agentur für Kommunikation GmbH, designed by Publik.
+Agentur für Kommunikation GmbH
```
Comment (BirgitBoss) [inline documentation/IDTA-01001/modules/ROOT/pages/annex/handling-constraints.adoc:12]: See above

**Review by BirgitBoss — CHANGES_REQUESTED** [review]
Looks good, only minor findings; see comments

File: documentation/IDTA-01001/modules/ROOT/pages/annex/backus-naur-form.adoc (hunk 1 above)
Comment (zrgt) [reply documentation/IDTA-01001/modules/ROOT/pages/annex/backus-naur-form.adoc]: Fixed in https://github.com/admin-shell-io/aas-specs/pull/452/commits/eb2845590da58d2a5ee077874855d437c0f39299

File: documentation/IDTA-01001/modules/ROOT/pages/annex/concepts-aas.adoc (hunk 2 above)
Comment (zrgt) [reply documentation/IDTA-01001/modules/ROOT/pages/annex/concepts-aas.adoc]: Fixed in https://github.com/admin-shell-io/aas-specs/pull/452/commits/eb2845590da58d2a5ee077874855d437c0f39299

File: documentation/IDTA-01001/modules/ROOT/pages/annex/handling-constraints.adoc (hunk 5 above)
Comment (zrgt) [reply documentation/IDTA-01001/modules/ROOT/pages/annex/handling-constraints.adoc]: Fixed in https://github.com/admin-shell-io/aas-specs/pull/452/commits/eb2845590da58d2a5ee077874855d437c0f39299

File: documentation/IDTA-01001/modules/ROOT/pages/annex/concepts-aas.adoc (hunk 3 above)
Comment (zrgt) [reply documentation/IDTA-01001/modules/ROOT/pages/annex/concepts-aas.adoc:92]: There is no space before == here

File: documentation/IDTA-01001/modules/ROOT/pages/annex/concepts-aas.adoc (hunk 4 above)
Comment (zrgt) [reply documentation/IDTA-01001/modules/ROOT/pages/annex/concepts-aas.adoc:117]: There is no space before == here

**Review by BirgitBoss — APPROVED** [review]

**Pull request merged**